conjur-oss/README.md

```diff
@@ -104,6 +104,39 @@ $  helm install \
    ./conjur-oss
 ```
 
+### Configuring Conjur Accounts
+
+By setting `account.create` to `true`, you can direct your Conjur
+container to create an admin account during startup. To retrieve the credentials
+for this account, perform the following commands:
+
+```sh-session
+export ACCOUNT_NAME=<Value for `account.name`>
+export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} \
+                                    -l "app={{ template "conjur-oss.name" . }},release={{ .Release.Name }}" \
+                                    -o jsonpath="{.items[0].metadata.name}")
+kubectl exec --namespace $CONJUR_NAMESPACE \
+              $POD_NAME \
+              --container=conjur-oss \
+              -- conjurctl role retrieve-key $ACCOUNT_NAME:user:admin
+```
+
+If you set `account.create` to `false`, or did not provide a value, an admin account will
+need to be created. To create an account, use the following commands:
+
+```sh-session
+export ACCOUNT_NAME=<Name for Conjur account to be created>
```

Comment:
Similar comments as for lines 114-122, I'd recommend using unexported variables
with values that have dashed descriptions, and remove the Helm template language
stuff, e.g.:
```
ACCOUNT_NAME=<conjur-account-name>
CONJUR_NAMESPACE=<conjur-namespace>
HELM_RELEASE=<helm-release>
export POD_NAME=$(kubectl get pods --namespace "$CONJUR_NAMESPACE" \
                                    -l "app=conjur-oss,release=$HELM_RELEASE" \
                                    -o jsonpath="{.items[0].metadata.name}")
kubectl exec --namespace "$CONJUR_NAMESPACE" \
              "$POD_NAME" \
              --container=conjur-oss \
              -- conjurctl account create "$ACCOUNT_NAME"